Got the following error:
```bash
Run /tmp/test/.venv/bin/python main.py test
Traceback (most recent call last):
  File "/home/pi/actions-runner/_work/sss/sss/main.py", line 3, in <module>
    from runners import simulator, kiosk, demo, test, utils
  File "/home/pi/actions-runner/_work/sss/sss/runners/simulator.py", line [7](https://github.com/NET-BYU/sss/runs/7138415273?check_suite_focus=true#step:6:8), in <module>
    import pygame
  File "/tmp/test/.venv/lib/python3.9/site-packages/pygame/__init__.py", line [8](https://github.com/NET-BYU/sss/runs/7138415273?check_suite_focus=true#step:6:9)3, in <module>
    from pygame.base import *  # pylint: disable=wildcard-import; lgtm[py/polluting-import]
ImportError: libSDL2-2.0.so.0: cannot open shared object file: No such file or directory
```

--- a/main.py
+++ b/main.py
@@ -1,6 +1,6 @@
 import click
 
-from runners import simulator, kiosk, demo, test, utils
+from runners import kiosk, demo, test, utils
 
 
 @click.group()
@@ -11,6 +11,9 @@ def cli():
 @cli.command(name="simulator")
 def run_simulator():
     """CLI command to run simulator."""
+
+    from runners import simulator  # pylint: disable=import-outside-toplevel
+
     simulator.run()
 
 

